substrate/frame/election-provider-multi-phase/src/benchmarking.rs

```diff
@@ -17,10 +17,9 @@
 
 //! Two phase election pallet benchmarking.
 
-use super::*;
-use crate::{unsigned::IndexAssignmentOf, Pallet as MultiPhase};
-use frame_benchmarking::account;
-use frame_election_provider_support::bounds::DataProviderBounds;
+use core::cmp::Reverse;
+use frame_benchmarking::{v2::*, BenchmarkError};
```

Comment:
```suggestion
use frame_benchmarking::v2::*;
```

`BenchmarkError` is already in `v2`